integration-test/atomic.config.js

```diff
@@ -0,0 +1,30 @@
+const cssPresetEnv = require("postcss-preset-env");
+const cssVariables = require("postcss-css-variables");
+const postcssFor = require("@utilitycss/postcss-for").default;
+const nested = require("postcss-nested");
+
+module.exports = {
+  packageScope: "@my-org",
+  electronsFolder: "electrons",
+  electronsModuleName: "@my-org/electrons",
+  atomsFolder: "atoms",
+  utilityConfigPath: "packages/atoms/utility.config.js",
+  bundleCSSPath: "packages/atoms/all/",
+  bundleCSSName: "atom",
+  additionalPlugins: {
```

Comment:
@sylvesteraswin plugins can now be injected from the consumer repo with whatever version and they arenot part of atomic node modules anymore

(we'll need to use this config to get results as before)